tanner/emulators/pad_oracle.py

```diff
@@ -0,0 +1,18 @@
+from tanner.utils import patterns
+
+
+class PadOracleEmulator:
+
+    def scan(self, value):
+        detection = None
+        if patterns.PAD_ORACLE_ATTACK.match(value):
+            detection = dict(name='pad_oracle', order=2)
+        return detection
+
+    def get_pad_oracle_results(self, attack_params):
```

Comment:
What is the purpose of returning the same input params as header?

Reply:
`attack_params` is a dictionary with many iterations whereas `header` is returning only the first iteration